include/engine/datafacade/internal_datafacade.hpp

```diff
@@ -662,6 +678,8 @@ class InternalDataFacade final : public BaseDataFacade
     }
 
     std::string GetTimestamp() const override final { return m_timestamp; }
+
+    bool GetUTurnsDefault() const override final { return m_profile_properties.allow_u_turn_at_via; }
```

Comment:
I don't think there are benefits for putting `final` on all those member functions, as the facade class itself is already marked `final`.